sources/nerdfonts/NerdfontsPL.ufo/glyphs/uniE_0B_6.glif

```diff
@@ -7,26 +7,20 @@ uniE0B6
 </note>
   <outline>
     <contour>
-      <point x="1200" y="-492" type="curve"/>
-      <point x="1200" y="-492" type="line"/>
-      <point x="1200" y="1990" type="line"/>
-      <point x="1003" y="1990"/>
-      <point x="804" y="1931"/>
-      <point x="602" y="1813" type="curve" smooth="yes"/>
-      <point x="200" y="1578"/>
-      <point x="0" y="1222"/>
-      <point x="0" y="749" type="curve" smooth="yes"/>
-      <point x="0" y="276"/>
-      <point x="201" y="-76"/>
-      <point x="602" y="-311" type="curve" smooth="yes"/>
-      <point x="805" y="-430"/>
-      <point x="1003" y="-492"/>
+      <point x="1200" y="-692" type="curve"/>
+      <point x="1200" y="-692" type="line"/>
+      <point x="1200" y="1790" type="line"/>
+      <point x="477" y="1790"/>
+      <point x="0" y="1297"/>
+      <point x="0" y="549" type="curve" smooth="yes"/>
+      <point x="0" y="-199"/>
+      <point x="477" y="-692"/>
```

Comment:
wow, this glyph got way simpler

Reply:
Yup. On this one and the one facing the other way, I removed some of the intermediary points, which were unnecessary holdovers from the TTF source file. It'd be nice to go through and clean up all the Nerd Fonts / PowerLine symbols, but that'd be quite a task.